src/ign_TEST.cc

```diff
@@ -86,13 +86,13 @@ class CmdLine : public ::testing::Test
 };
 
 // See https://github.com/ignitionrobotics/ign-gui/issues/75
-// See https://github.com/gazebosim/gz-gui/issues/415
-TEST_F(CmdLine, DETAIL_IGN_UTILS_ADD_DISABLED_PREFIX(list))
+TEST_F(CmdLine, IGN_UTILS_TEST_ENABLED_ONLY_ON_LINUX(list))
 {
   // Clear home if it exists
   common::removeAll(this->kFakeHome);
 
-  EXPECT_FALSE(common::exists(this->kFakeHome));
+  // Can't EXPECT_FALSE here, see https://github.com/gazebosim/gz-gui/issues/415
+  common::exists(this->kFakeHome);
```

Comment:
Is it useful/needed to check if it exists if we are not checking that information?